client/lib/products-values/constants.js

```diff
@@ -25,3 +35,12 @@ export const JETPACK_BACKUP_PRODUCT_NAMES = {
 	[ PRODUCT_JETPACK_BACKUP_REALTIME ]: 'Real-Time Backups',
 	[ PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY ]: 'Real-Time Backups',
 };
+
+export const JETPACK_BACKUP_UPGRADE_MAP = {
```

Comment:
@enejb I'm wondering whether this map should be 'reversed'. What I mean by that is to have each key representing a (Jetpack Backup) product and then the value would be an array of other products/plan that you can upgrade from, e.g.:

```js
export const JETPACK_BACKUP_UPGRADE_MAP = {
	[ PRODUCT_JETPACK_BACKUP_REALTIME ] : [
		PRODUCT_JETPACK_BACKUP_DAILY,
		PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
		PLAN_JETPACK_PREMIUM,
		PLAN_JETPACK_PERSONAL,
		PLAN_JETPACK_PREMIUM_MONTHLY,
		PLAN_JETPACK_PERSONAL_MONTHLY,
	],
	[ PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY ] : [
		...
	],
}
```

I think it would be easier to look up and manage the list this way.

Reply:
I though about this too. I kind of like how it currently works. 
Since it ensures that there is only 1 -> 1 mapping. 

I could see how for example we could put the plan that that should be updated into 2 different lists. 

Also the loop up is much easier. We don't have to loop though all the keys and the `JETPACK_BACKUP_UPGRADE_MAP` map.

Reply:
Makes sense, let's keep it as is 👍